lib/msal-browser/docs/logout.md

```diff
@@ -86,6 +86,52 @@ await msalInstance.logoutPopup({
 });
 ```
 
+## Front-channel logout
```

Comment:
Can we add a TOC to this file? There are quite a few sections and some things might get lost if we don't have a reference to them at the top of the file.

Reply:
TOC are now added by Github automatically to all md files: https://github.blog/changelog/2021-04-13-table-of-contents-support-in-markdown-files/